### ISSUE TITLE
Creating a single file in /etc/openhab/rules generates two “Loading model” messages

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-core#3006]
When I move a file to `/etc/openhab/rules`, openhab logs:
```
2022-06-18 09:14:39.605 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading model 'system_started.rules'
2022-06-18 09:14:40.799 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading model 'system_started.rules'
```
The file has not existed in the destination, before I copied it.

The problem report is, that the above log message is emitted twice, but it shall appear only once.

**Comment by wborn** [comment]
It's probably because the `WatchService` receives 2 events from the OS whenever a file is created. https://github.com/openhab/openhab-core/pull/3004 might improve this because it uses a library which can prevent handling such duplicate events.

See: https://github.com/gmethvin/directory-watcher#configuration

**Comment by J-N-K** [comment]
Duplicate of #671
